docs/query-syntax.md

```diff
@@ -73,6 +77,8 @@ Like in [Streaming Tail](#streaming-tailing), Timeplus continuously monitors new
 continuously does **incremental** count aggregation. Whenever the specified delay interval is up, project the current aggregation result
 to clients.
 
+\
```

Comment:
same issue

Reply:
In general, we should not care too much about line breaks in documentation. The martin/etc doesn't matter in documentations. For me, if I really have to add new lines, I will use <br>